### PR TITLE
refactor(rxPermalink): icon on left

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-856

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM
- [x] QE LGTM (no change to javascript)

# Before

permalinks-before

permalinks-before-2

# After

permalinks-after

permalinks-after-2

### CODE REVIEW

**Comment by CITguy** [comment]
Design Team:

rxpermalink-left-icon

File: demo/less/_rxPermalink.less [hunk 1]
```diff
@@ -1,8 +1,16 @@
 rx-permalink {
   .permalink {
+    position: relative;
     .icon {
+      top: 0px;
```
Comment (ericcarraway) [inline demo/less/_rxPermalink.less]: I don't know if this should be `top: 0;` instead. 

This guideline from the docs may be outdated or inapplicable here.

screen shot 2016-11-18 at 1 32 31 pm

http://rackerlabs.github.io/encore-ui/#/guides/css-styleguide

**Comment by halkon** [comment]
Vidyo LGTM

**Comment by ElementE** [comment]
Design LGTM

**Pull request merged**